src/coreComponents/physicsSolvers/fluidFlow/wells/WellControls.cpp

```diff
@@ -36,7 +37,13 @@ WellControls::WellControls( string const & name, Group * const parent )
   m_targetPhaseName( "" ),
   m_useSurfaceConditions( 0 ),
   m_surfacePres( 0.0 ),
-  m_surfaceTemp( 0.0 )
+  m_surfaceTemp( 0.0 ),
+  m_targetTotalRateTableName( "" ),
+  m_targetPhaseRateTableName( "" ),
+  m_targetBHPTableName( "" ),
```

Comment:
String members don't need initializers as they have default constructors. I also prefer that default values to all non-default-initialized members are assigned at declaration rather than constructor, but I don't suggest changing them at this time (because there's too much code in GEOSX that uses this style).